Failed to Compile --Learn Apollo in Windows environment

I started with the Learn Apollo tutorial, using Windows Pc and got the following error message after npm start command.

Failed to compile.

Error in ./~/styled-components/lib/vendor/postcss/css-syntax-error.js
Module not found: Error: Cannot resolve module 'supports-color' in D:\apollo\react\pokedex\exercise-01\node_modules\styled-components\lib\vendor\postcss
 @ ./~/styled-components/lib/vendor/postcss/css-syntax-error.js 9:21-46

--- a/package.json
+++ b/package.json
@@ -47,7 +47,8 @@
     "glamor": "^2.20.12",
     "inline-style-prefixer": "^2.0.5",
     "is-function": "^1.0.1",
-    "is-plain-object": "^2.0.1"
+    "is-plain-object": "^2.0.1",
+    "supports-color": "^3.1.2"
   },
   "devDependencies": {
     "@types/react": "^0.14.55",
@@ -99,7 +100,6 @@
     "rollup-plugin-replace": "^1.1.1",
     "rollup-plugin-uglify": "^1.0.1",
     "rollup-plugin-visualizer": "^0.1.5",
-    "supports-color": "^3.1.2",
     "tslint": "^4.3.1",
     "typescript": "^2.1.5"
   },

